src/argp.c

```diff
@@ -592,8 +591,8 @@ int run_command(cli_command *commands, int cmd, int argc, char **argv)
 
 	/* execute command */
 	result = commands[cmd].handle();
-	ftime(&t1);
-	timespan = (1000 * (t1.time - t0.time) + t1.millitm - t0.millitm);
+	clock_gettime(CLOCK_REALTIME, &t1);
+	timespan = (1000 * (t1.tv_sec - t0.tv_sec) + t1.tv_nsec - t0.tv_nsec);
```

Comment:
timespan calculation uses incorrect multiplier (1000) for clock_gettime() method -- ftime() returned miliseconds while clock_gettime() returns nanoseconds. Please fix it

Reply:
You're right of course, fixed in latest push.

Insert some quote about coding tired and reviewing awake. ;)